Answers.md

```diff
@@ -0,0 +1,6 @@
+1. 	If i had to describe semantic HTML to the next cohort of students, I would simply say that semantic tags are ways that "we" the developer better lable our HTML code, so that it can be read by you and your fellow developers more easily. Also search engines use semantic tags to display a lot of information about your website so tagging them correctly is important.
+2.	display:block; places a specifically sectioned group of things on a webpage in their own block taking up a whole line of HTML unless coded otherwise 
```

Comment:
Yup pretty much.

block - its width is defined by the width of its container
inline - its width is defined by what's inside if it